README.md

```diff
@@ -61,6 +61,17 @@ Build binary:
 make build
 ```
 
+Run Unit/Integ Tests:
+```console
+make test
+```
+(see internal/README.md for explanation of tests)
+
+Run tests against kind cluster and live controller:
+```console
+make e2e
```

Comment:
I believe it will fail without `UPTEST_EXAMPLE_LIST` variable to be set, see https://github.com/aviatrix/crossplane-provider-aviatrix/blob/main/Makefile#L169

Also, it needs to be clarified which examples from https://github.com/aviatrix/crossplane-provider-aviatrix/tree/main/examples are ready for uptest. What is the expectation there?

Reply:
All files in the examples directory work with uptest.  I don't promote a resource until it has a functional example.  

I'm still not certain what the proper value for UPTEST_EXAMPLE_LIST is.  Can you point me to documentation on that env var?

Reply:
> All files in the examples directory work with uptest. I don't promote a resource until it has a functional example.

That is just perfect! Please just drop a line to the doc about it so provider users can have good confidence in  `examples` directory.

> I'm still not certain what the proper value for UPTEST_EXAMPLE_LIST is. Can you point me to documentation on that env var?

It looks like we miss the doc like https://github.com/upbound/provider-aws/blob/main/Makefile#L173-L175 in the provider template; good catch. I've just filed https://github.com/upbound/upjet-provider-template/issues/47 to track it long-term.

So ` - UPTEST_EXAMPLE_LIST, a comma-separated list of examples to test` , the idea is to be able to invoke the e2e target targeting the specific examples, e.g.

```
make e2e UPTEST_EXAMPLE_LIST="examples/gatewaypeering.yaml,examples/transit-gateway.yaml,examples/transit-gateway2.yaml"
```
This way you can adjust e2e test run according to the desired scenario, e.g. scoping the heavy uptest run to a specific PR under test.